data/hai/hai missions.txt

```diff
@@ -144,7 +230,7 @@ mission "First Contact: Unfettered"
 				`	"I'm sorry, I don't know the secret of the jump drive. No human being does."`
 				`	"If I ever have an extra jump drive to sell, I will bring it to you."`
 					goto sell
-			`	"Then leave us alone," it says, and it walks off.`
+			`	"Then leave us alone," it says, and it walks off."`
```

Comment:
Not sure why you've added this. (This same error exists in the new conversation.)
```suggestion
			`	"Then leave us alone," it says, and it walks off.`
```

Reply:
Copy paste error, I've just been working through these, since they weren't flagged until I tried to PR them here. Sorry.